mongoexport/options_test.go

```diff
@@ -106,3 +109,60 @@ func TestParseOptions(t *testing.T) {
 		}
 	})
 }
+
```

Comment:
I'm not sure we need to repeat the same tests in every one of the tools. Is there some good way we could centralize these tests into m-c-t?

Reply:
That's a good question. Each tool has a `ParseOptions()` function. These functions do tool-specific parsing and error handling. If we wanted to properly test all the errors in m-t-c, we would have to push some of that tool-specific logic into m-t-c. I actually started implementing this PR in that way, but eventually decided on the current approach. My reasoning was:

1. We should limit the amount of tool-specific code in m-t-c. It's meant to be a library of common functionality. If m-t-c performs different logic for each tool, it will become hard to reason about the functionality of tools when just looking at the tools repo.
2. It's more prone to regression if the code path is still `main()`->`ParseOptions()`->`options.ParseArgs()`. If we're only testing the m-t-c portion (`options.ParseArgs`), it won't catch regressions that could arise in `ParseOptions()`.

I agree it's not ideal, but I don't think there's a quick and easy way to centralize these tests.